cases.ts

```diff
@@ -163,7 +163,7 @@ export const rule5Tree: ASTNode = {
           args: [
             {
               type: 'value',
-              value: ['investment_company', 'custodial_institution', 'depositary_institution', 'specified_insurance_company'],
+              value: "['investment_company', 'custodial_institution', 'depositary_institution', 'specified_insurance_company']",
```

Comment:
With the new property `dataType` we could preserve the array.

Reply:
to preserve the arrays, i have undo the changes and implemented autocomplete of quotes